rotkehlchen/db/accounting_rules.py

```diff
@@ -117,30 +131,85 @@ def update_accounting_rule(
                     ' but it was not found',
                 )
 
+    def add_overwritten_setting(
+            self,
+            rule_identifier: int,
+            rule_property: Literal['count_entire_amount_spend', 'count_cost_basis_pnl'],
+            setting_name: Literal['include_gas_costs', 'include_crypto2crypto'],
+            is_active: bool = True,
+    ) -> None:
+        """
+        Store in the database a rule overwritten property that links it to a setting in the
```

Comment:
You use the term "overwrite" here alot. This is probably not the best term. Perhaps just use "link_rule_to_setting" everywhere.